### DISCUSSION TITLE
Touch @ Wemos R32(ESP32)

### DISCUSSION THREAD

**Discussion by engvidal** [Bodmer/TFT_eSPI#1211] [Q&A - Touch]
Hi
I did all hardware modifications in my Wemos R32(ESP32):
IO15 wired to IO35
IO33 wired to IO34
IO32 wired to IO36

I am using your setup model:
**https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup14_ILI9341_Parallel.h**

But in this discussion you said to use a pin to CS of touch.

_If you define a pin for the touch chip select then it should be OK:
https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setup.h#L97_

In your setup file what is the correct pin to CS of touch?

Thanks

**Comment by Pablo2048** [comment]
This would not work - GPIO > 33 are input only!!! (see the documentation for ESP32)...

**Reply by xPrject** [reply]
33 is input output

ref: esp32 documentation

IO32 8 I/O
IO33 9 I/O
IO34 6 I 
IO35 7 I

**Reply by Pablo2048** [reply]
Lowest GPIO# used in the question is 34, so it's as I said...

**Comment by xPrject** [comment]
> Hi I did all hardware modifications in my Wemos R32(ESP32): IO15 wired to IO35 IO33 wired to IO34 IO32 wired to IO36
> 
> I am using your setup model: **https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup14_ILI9341_Parallel.h**
> 
> But in this discussion you said to use a pin to CS of touch.
> 
> _If you define a pin for the touch chip select then it should be OK: https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setup.h#L97_
> 
> In your setup file what is the correct pin to CS of touch?
> 
> Thanks

wire IO4 to IO39

**Reply by hadypranoto** [reply]
I also following that instruction..and still not working. Im missing someting here?

**Reply by xPrject** [reply]
Ill put togethee a wemos later ill let you know imma try it myseld if you could post pics of what you did that might help
